src/EventTrackerService.php

```diff
@@ -40,19 +44,51 @@ class EventTrackerService {
    */
   private $eventStorage;
 
+  /**
+   * The current user.
+   *
+   * @var \Drupal\Core\Session\AccountInterface
+   */
+  protected $currentUser;
```

Comment:
I think the new dependencies should be `private` to be consistent with the other members.